components/builder-depot/src/server.rs

```diff
@@ -1142,6 +1166,13 @@ pub fn router(depot: Depot) -> Result<Chain> {
                 XHandler::new(upload_package).before(basic.clone())
             }
         },
+        package_schedule: post "/pkgs/schedule/:origin/:pkg/:version/:release" => {
+            if depot.config.insecure {
+                XHandler::new(schedule_package)
+            } else {
+                XHandler::new(schedule_package).before(basic.clone())
```

Comment:
This is totally cool here for now since it's just for debugging, as we add endpoints that we want to hide until the build service is ready - you can protect them with the `Authentication` value bound to`worker` above. It'll ensure that the caller has the proper feature flag enabled